tensorflow/workspace.bzl

```diff
@@ -184,6 +184,17 @@ def tf_repositories(path_prefix = "", tf_repo_name = ""):
         ],
     )
 
+    tf_http_archive(
+        name = "fmtlib",
```

Comment:
We try to minimize the use of external dependencies. Any chance this job could be done by a standard library?

Reply:
We have utilities in `strings::Printf` that should do this.

Reply:
I also noticed from the implementation of `tf.as_string` that the `strings::Printf` method can achieve exactly the same function. However, in my training scenario, because my model contains a large number of string conversions, this method has become a bottleneck for further performance breakthroughs. Considering that the performance of this implementation (i.e. `vsnprintf` in fact) is relatively poor while input swelling, and according to this [benchmark](http://www.zverovich.net/2020/06/13/fast-int-to-string-revisited.html), **`fmt::to_string` is 4x faster than `sprintf`**, so I introduced `fmt::to_string` and submitted this pr. 

P.S. I have reorganized the relevant instructions and benchmarks [here](https://github.com/tensorflow/tensorflow/pull/45466#issue-534189832), could you please have a look at it again?

Reply:
Maybe we can learn something from `fmt`'s implementation. The issue is that adding new dependencies might incur larger costs than the performance gain for a very infrequent scenario.

Reply:
Instead of using `fmt` as a new third-party dependency, if we can migrate the implementation code of `fmt::to_string` to tensorflow (e.g. number.cc) alone, may it be an acceptable solution? Because in fact I only used one function in that library. WDYT? @mihaimaruseac

Reply:
That could work.

Alternatively, if you want to ue `fmt::to_string` there is the option of making this a custom op that you load when needed (from outside this repo)